File: HFS-BE/HFS-BE/Models/Shipper.cs
using System;
using System.Collections.Generic;

namespace HFS_BE.Models
{
    public partial class Shipper
    {
        public Shipper()
        {
            Invitations = new HashSet<Invitation>();
            OrderProgresses = new HashSet<OrderProgress>();
            Orders = new HashSet<Order>();
        }

        public string ShipperId { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string? Gender { get; set; }
        public DateTime? BirthDate { get; set; }
        public string Email { get; set; } = null!;
        public string? PhoneNumber { get; set; }
        public byte[] PasswordSalt { get; set; } = null!;
        public byte[] PasswordHash { get; set; } = null!;
        public bool? IsOnline { get; set; }
        public string? ManageBy { get; set; }
        public bool? ConfirmedEmail { get; set; }
        public byte Status { get; set; }
        public string? RefreshToken { get; set; }
        public DateTime? RefreshTokenExpiryTime { get; set; }
        public DateTime? CreateDate { get; set; }
        public string? Note { get; set; }
        public bool? IsPhoneVerified { get; set; }
        public string? OtpToken { get; set; }
        public int? OtpTokenExpiryTime { get; set; }
        public string? IdcardNumber { get; set; }
        public string? IdcardFrontImage { get; set; }
        public string? IdcardBackImage { get; set; }

        public virtual Seller? ManageByNavigation { get; set; }
        public virtual ICollection<Invitation> Invitations { get; set; }
        public virtual ICollection<OrderProgress> OrderProgresses { get; set; }
        public virtual ICollection<Order> Orders { get; set; }
    }
}

File: HFS-FE/src/app/modules/admin-routing-module/models/Shipper.ts
export class Shipper {
  shipperId: string;
  shipperName: string;
  gender: string;
  birthDate: Date;
  email: string;
  phoneNumber: string;
  avatar: string;
  isPhoneVerified:boolean;
  confirmedEmail:boolean;
  isBanned: boolean;
  manageBy:string
  status:number;
  note:string;
  createDate:Date;
  idcardNumber:string;
  imageBase64Id1:string;
  imageBase64Id2:string;
  imagesBase64: ShipperImageBase64[] = null;
}
export interface ShipperImageBase64 {
  imageId: number;
  imageBase64: string;
  name: string;
  size: string;
}
export class  HistoryBanShipper{
  banShipperId: number;
  reason:string;
  createDate:Date;
}
export class BanShipper {
  shipperId: string;
  reason:string;
  isBanned: boolean;
}
export class ActiveShipper1 {
  shipperId: string;
  isVerified:boolean;
}
export class ActiveShipper {
  shipperId: string;
  note:string;
  status: number;
}
